src/models/FunctionTools.h

```diff
@@ -50,5 +53,6 @@ class TemplateFunction {
     PTRef getBody() const { return tr_body; }
     const vec<PTRef>& getArgs() const { return signature.getArgs(); }
     void updateBody(PTRef new_body) { tr_body = new_body; }
+    bool isInterpreted() const { return signature.getSymbolConfig().isInterpreted; }
```

Comment:
You're right.  It feels silly to hard-code that all concretisations of template functions are non-interpreted, but I don't really have a use case when this would not be so.  I'll remove the functionality for now.